packages/@ama-sdk/core/package.json

```diff
@@ -76,6 +77,9 @@
     "@angular/common": {
       "optional": true
     },
+    "@o3r/core": {
+      "optional": true
```

Comment:
is it really optional ?
regardless, `@o3r/core` should also be added in the devDependencies generated by the @ama-sdk/schematics:typescript-shell https://github.com/AmadeusITGroup/otter/blob/release/10.0.0-next/packages/%40ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template#L79

Reply:
I think it is optional since it is only used to import types (`Logger` from `@o3r/core`)